lora/radio.go

```diff
@@ -0,0 +1,13 @@
+package lora
+
+type Radio interface {
+	Reset()
+	Tx(pkt []uint8, timeoutMs uint32) error
+	Rx(timeoutMs uint32) ([]uint8, error)
+	SetFrequency(freq uint32)
+	SetIqMode(mode uint8)
+	SetCodingRate(cr uint8)
+	SetBandwidth(bw uint8)
+	SetCrc(enable bool)
+	SetSpreadingFactor(sf uint8)
+}
```

Comment:
We could add LoraConfig() , SetSyncWord , SetTxPower, SetPreamble ... 
as well ....

Reply:
We can add in future PRs as needed.